sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1764,6 +1764,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ARITHMETIC_OPERATION_OVERFLOW_CHECK = buildConf("spark.sql.arithmetic.checkOverflow")
```

Comment:
`check` is vague here, how about `spark.sql.arithmeticOperations.failOnOverFlow`

Reply:
thanks for the suggestion, I'll update accordingly